src/mds/MetricsHandler.cc

```diff
@@ -0,0 +1,241 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#include "common/debug.h"
+#include "common/errno.h"
+
+#include "messages/MMDSMetrics.h"
+
+#include "MDSRank.h"
+#include "SessionMap.h"
+#include "MetricsHandler.h"
+
+#define dout_context g_ceph_context
+#define dout_subsys ceph_subsys_mds
+#undef dout_prefix
+#define dout_prefix *_dout << __func__ << ": mds.metrics"
```

Comment:
Here we may need to add a space at the end, or it will be looks like:

`out/mds.c.log:81256:2019-12-19T06:19:23.530-0500 7fabf99d0700 20 handle_client_metrics: mds.metricshandle_client_metrics: session=0x560d23892880`